assets/app/scripts/controllers/overview.js

```diff
@@ -354,6 +362,18 @@ angular.module('openshiftConsole')
       Logger.log("builds (subscribe)", $scope.builds);
     }));
 
+    // Show the "Get Started" message if the project is empty.
+    function updateShowGetStarted() {
+      // Check if there are any services, pods, or deployment configs.
+      var projectEmpty =
```

Comment:
reframe this as `hasContent` so you can short-circuit. `angular.equals` looks scary and expensive... wondering if `$filter('hashSize')(...) == 0` could work instead